subsystems/drivebase/SwerveDrive.java

```diff
@@ -311,8 +327,24 @@ public void mustangPeriodic() {
         Logger.recordOutput(DRIVEBASE_PITCH, getPitch());
         Logger.recordOutput(DRIVEBASE_ROLL, getRoll());
 
+       
+
+        if (kConfig.kDriveMotorType.equals(Motor_Type.KRAKEN_X60)) {
+            emaCurrent = currentAlpha * ((TalonFX) mModules[0].getDriveMotor()).getSupplyCurrent() + (1 - currentAlpha) * emaCurrent; // weighted current average of front-right drive motor
+            if (emaCurrent > 30) { // TODO tune this? what should be the threshold?
```

Comment:
The trouble with a weighted average like this is that it's tough to get a sense for what it means. Which just makes this TODO even harder to reason about. Did you think about doing a rolling window? It wouldn't be hard to keep a buffer that we maintain for 5 seconds or something and we track the average of the drive current draw over the course of those 5 seconds. If it exceeds some threshold (120 amps? 100 amps?) we cut the current limit